feat(A1): convert to wide format

cmdoret opened the pull request:
This PR converts A1 from tidy to wide format. Namely, it creates a new column for each level of the `geraet_oder_untersuchung` and inserts corresponding values from the `anzahl` column. Here is a glimpse at the table structure:

Before:

```
Rows: 7,776
Columns: 6
$ genutzte_infrastruktur   <chr> "Angiographiegeräte", "Angiographiegeräte", "…
$ geraet_oder_untersuchung <chr> "Anzahl Geräte", "Anzahl Geräte", "Anzahl Ger…
$ jahr                     <chr> "2013", "2014", "2015", "2016", "2017", "2018…
$ anzahl                   <dbl> 137, 138, 146, 143, 136, 136, 128, 141, 145, …
$ spatialunit_ontology     <chr> "Country", "Country", "Country", "Country", "…
$ spatialunit_uid          <chr> "0_CH", "0_CH", "0_CH", "0_CH", "0_CH", "0_CH…
```
After:

```
Rows: 1,944
Columns: 8
$ genutzte_infrastruktur    <chr> "Angiographiegeräte", "Angiographiegeräte", …
$ jahr                      <chr> "2013", "2014", "2015", "2016", "2017", "201…
$ spatialunit_ontology      <chr> "Country", "Country", "Country", "Country", …
$ spatialunit_uid           <chr> "0_CH", "0_CH", "0_CH", "0_CH", "0_CH", "0_C…
$ anzahl_gerate             <dbl> 137, 138, 146, 143, 136, 136, 128, 141, 145,…
$ untersuchungen_total      <dbl> 105713, 116967, 119395, 121854, 118025, 1209…
$ ambulante_untersuchungen  <dbl> 50540, 52212, 55384, 48955, 52134, 53721, 58…
$ stationare_untersuchungen <dbl> 54383, 62891, 62205, 68094, 65035, 66594, 62…
```

And a screenshot of what the table would look like after this PR:

![image](https://github.com/statistikZH/statbotData/assets/22558602/ca374c3d-aa3c-4cfd-8f85-da460f4a8915)

sabinem reviewed (approved):
Your code looks great. 
I would just suggest to rename a view columns: 

- `spatialunit_ontology` -> I think we can drop this column as we have the foreign key now.
- `untersuchungen_total` -> `anzahl_untersuchungen`
- `ambulante_untersuchungen` -> `anzahl_ambulante_untersuchungen`
- `stationare_untersuchungen` -> `anzahl_stationare_untersuchungen`

I will approve the PR, but if you agree with my suggestions regarding the columns you can do them.

sabinem commented on DESCRIPTION:
```diff
@@ -1,22 +1,23 @@
     dplyr,
     googlesheets4,
     magrittr,
+    stringdist,
```
I couldn't install this package. On google I found this: https://github.com/markvanderloo/stringdist/issues/84
This shouldn't prevent us from using it though. I just wanted to mention this.

cmdoret replied:
Based on this comment https://github.com/velocyto-team/velocyto.R/issues/57#issuecomment-506671728, it looks like an issue with llvm+R on MacOS.

philbosch reviewed (commented):
Agree with the proposed changes of @sabinem regarding the variable names. 
Still up to debate and decide by ZHAW/@nooralahzadeh if we should split the table and supply one table just with the amount of "Geräte".

cmdoret commented:
Thanks @sabinem, I updated the column names and dropped the unneeded spatialunits_ontology. 

```
> dplyr::glimpse(ds$data)
Rows: 1,944
Columns: 7
$ genutzte_infrastruktur           <chr> "Angiographiegeräte", "Angiographiege…
$ jahr                             <chr> "2013", "2014", "2015", "2016", "2017…
$ spatialunit_uid                  <chr> "0_CH", "0_CH", "0_CH", "0_CH", "0_CH…
$ anzahl_gerate                    <dbl> 137, 138, 146, 143, 136, 136, 128, 14…
$ anzahl_untersuchungen_total      <dbl> 105713, 116967, 119395, 121854, 11802…
$ anzahl_ambulante_untersuchungen  <dbl> 50540, 52212, 55384, 48955, 52134, 53…
$ anzahl_stationare_untersuchungen <dbl> 54383, 62891, 62205, 68094, 65035, 66…
```
